src/Authentication/Actions/Email2FA.php

```diff
@@ -70,13 +75,17 @@ public function handle(IncomingRequest $request)
 
         // Send the user an email with the code
         helper('email');
-        $email = emailer();
-        $email->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
+        $emailer = emailer();
+        $ret     = $emailer->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
```

Comment:
What is "ret"? If this is short for "return" can you just use the whole word?

Reply:
Yes, and fixed!